CHANGELOG.md

```
@@ -1,50 +1,33 @@
 ## [2.3.1](https://github.com/VilnaCRM-Org/user-service/compare/v2.3.0...v2.3.1) (2025-02-04)
```

markdownlint-cli2 (0.17.2)

First line in a file should be a top-level heading
null

(MD041, first-line-heading, first-line-h1)